owncloud/owncloud.py

```diff
@@ -39,41 +36,14 @@ def __init__(self, share_id, target_file, link, token):
 
     def __str__(self):
         return 'PublicShare(id=%i,path=%s,link=%s,token=%s)' % \
-               (self.share_id, self.target_file, self.link, self.token)
-
-
-class UserShare():
```

Comment:
Yeah, this is the problem with plain files.
Your PR would delete the latest code.

Best would be to apply your changes manually (add them inside the code) instead of replacing the whole file.

The goal is that this view here only shows whatever you changed.
Please use the actual latest version of owncloud.py and apply your changes on top of that.

Thanks.